src/test/resources/features/prodDescription.feature

```diff
@@ -0,0 +1,23 @@
+@ProductDesc
+Feature: Amazon health check
+  
+  @tag1
+  Scenario: Product description for amazon  %$45
```

Comment:
Please check the scenario line. Its having "%$45". Please remove it.

Reply:
I made changes at scenario line